content/pages/team.md

```diff
@@ -1,4 +1,75 @@
-Title: Team
+Title:
 Slug: team
 Author: Nolan Nichols <http://orcid.org/0000-0003-1099-3328>
 Summary: Team members of the SIBIS research group
+#Team
+---
+<div>
+	<img src='../images/team/sara.png' hspace='20'>
+		<strong>Sara Benito Arce</strong>,
+		<b>Biomedical Software Engineer</b>
+</div>
+---
```

Comment:
For example, this:

```
![Alt Sara Benito](https://raw.githubusercontent.com/sibis-platform/sibis-platform.github.io/c339e022d4be01a03823da4816d5d366d22261ef/content/images/team/sara.png)
__Sara Benito Arce__
_Biomedical Software Engineer_
```

Produces this:

![Alt Sara Benito](https://raw.githubusercontent.com/sibis-platform/sibis-platform.github.io/c339e022d4be01a03823da4816d5d366d22261ef/content/images/team/sara.png)
__Sara Benito Arce__
_Biomedical Software Engineer_